cluster-autoscaler/cloudprovider/gce/mig_info_provider.go

```diff
@@ -158,12 +175,20 @@ func (c *cachingMigInfoProvider) findMigWithMatchingBasename(instanceRef GceRef)
 }
 
 func (c *cachingMigInfoProvider) fillMigInstances(migRef GceRef) error {
+	if val, ok := c.migInstancesLastRefreshedInfo[migRef.String()]; ok {
+		// do not regenerate MIG instances cache if last refresh happened recently.
```

Comment:
Maybe log info that we're serving stale data from cache?

Reply:
done